django_app/redbox_app/redbox_core/views.py

```diff
@@ -108,22 +108,22 @@ def upload_view(request):
                 )
                 file.save()
                 # TODO: update improved File object with elastic uuid
-            except ValueError as value_error:
-                errors.append(value_error.args[0])
 
-            try:
-                api.upload_file(settings.BUCKET_NAME, uploaded_file.name, request.user)
+                api.upload_file(settings.BUCKET_NAME, file.name, request.user)
```

Comment:
just for my understanding, there will be separate embedding for each duplicate file name?

Reply:
There should be. The uniqueness of the `name` property is tested in the line: 
`assert File.objects.order_by("-created_at")[0].name != File.objects.order_by("-created_at")[1].name`

Based on Ell's comment, I'll create a new property (`unique_name`) for sharing to core_api, so that `name` is just the uploaded name the user has seen.

Reply:
thx for the explanation :+1: